protostar/protostar_cli.py

```diff
@@ -39,7 +39,6 @@ def __init__(
         self._version_manager = version_manager
         self._start_time = start_time
         self._project_cairo_path_builder = project_cairo_path_builder
-
```

Comment:
Unnecessary change outside scope of this PR

Reply:
The change is still there ;)

Reply:
Oh there are two of them